src/components/staking/Summary.svelte

```diff
@@ -55,16 +55,24 @@
               votingClass = "text-red";
               voteKeyword = "cannot_votes";
             } else {
-              if(oldestValidLock && $stakingData.proposals[0].block.timestamp < Number(oldestValidLock.lockedAt)) {
+              let proposal = get($stakingData.proposals, 0);
+              try {
+                if(oldestValidLock && proposal.block.timestamp < Number(oldestValidLock.lockedAt)) {
```

Comment:
```
get($stakingData, 'proposals[0].block.timestamp', false)
```

Using get with a default value enableß you to avoid breaking the flow when accessing any of the properties.

https://lodash.com/docs/4.17.15#get

Reply:
changed as requested